src/HOCs/RoutingByContextAuth.js

```diff
@@ -10,7 +10,7 @@ const RoutingByContextAuth = allowedRoles => WrappedComponent => props => {
     <AuthUserContext.Consumer>
       {userClaims => {
         if (userClaims != null) {
-          for (let i = 0; i < userClaims.length; i++) {
+          for (let i = 0; i < userClaims.length; i += 1) {
```

Comment:
This is the same right?

Reply:
Yeah it's the same, it's just that the linter made me change from i++ to i += 1 lol